src/pybind/ceph_argparse.py

```diff
@@ -1073,7 +1073,7 @@ def validate_command(sigdict, args, verbose=False):
                 break
 
         if not found:
-            print('no valid command found; 10 closest matches:', file=sys.stderr)
+            print('no valid command found; %d closest matches:'%min(len(bestcmds),10), file=sys.stderr)
```

Comment:
Keeping the style consistent would be better. how about:
```
print('no valid command found; {0} closest matches:'.format(min(len(bestcmds), 10)), file=sys.stderr)
```

Reply:
Ok, fair enough. Thanks for your review.